lib/utils.js

```diff
@@ -247,30 +247,45 @@ const deserializeContext = (headers = {}, prefix = 'podium') => {
 const template = ({
     head = '',
     body = '',
+    footer = '',
     encoding = 'utf-8',
     locale = 'en-US',
     title = '',
-    js = '',
-    css = '',
-} = {}) => `<!doctype html>
+    js = [],
+    css = [],
+} = {}) => {
+    let scripts = js;
+    let styles = css;
+
+    const buildPreloadTag = ({ value = '' }) =>
+        `<link rel="preload" href="${value}" as="script"></link>`;
+    const buildScriptTag = ({ value = '' }) =>
+        `<script src="${value}" defer></script>`;
+    const buildCSSLinkTag = ({ value = '' }) =>
+        `<link media="all" rel="stylesheet" type="text/css" href="${value}">`;
```

Comment:
the `media` attribute has the value `all` by default, so I don't think we should add it.

For the same reason we don't add attributes with defaults, like a `type="text/javascript"` to the `<script>` tags 🙂